modules/nextflow/src/main/groovy/nextflow/k8s/K8sConfig.groovy

```diff
@@ -169,6 +169,11 @@ class K8sConfig implements Map<String,Object> {
         podOptions
     }
 
+    boolean fetchNodeName() {
+        log.info( target.toString() )
```

Comment:
This would be shown in the console. it's really needed I would even decrease to trace

Reply:
Well, this was for testing the behavior of `as boolean` and should not have been committed.